packages/icons/src/trimet-mod-leg-icon.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+import LegIcon from "./leg-icon";
+import TriMetModModeIcon from "./trimet-mod-mode-icon";
+
+const TriMetModLegIcon = props => (
+  <LegIcon ModeIcon={TriMetModModeIcon} {...props} />
+);
```

Comment:
Let's consider deleting in favor of `trimet-leg-icon`. It seems silly to have multiple TriMet leg icons.

Reply:
I have renamed the `mod` set to `standard`.